axelor-crm/src/main/java/com/axelor/apps/crm/exception/CrmExceptionMessage.java

```diff
@@ -140,4 +140,7 @@ private CrmExceptionMessage() {}
 
   public static final String CRM_DEFAULT_PARTNER_STATUS_MISSING = /*$$(*/
       "Please fill default partner status in App CRM configuration." /*)*/;
+
+  public static final String CRM_EMAIL_DOMAIN_ALREADY_EXISTS = /*$$(*/
+      "We found the following similar leads/contacts/Tiers in the system, do you still want to create this lead ?" /*)*/;
```

Comment:
Please change this message to `We found the following similar leads/contacts/partners in the system, do you still want to create this lead?`